File: logica-de-programacao/secao-9/exercicios/menor_de_tres.c
#include <stdio.h>
#include <math.h>


void limpar_entrada() {
    char c;
    while ((c = getchar()) != '\n' && c != EOF) {}
}


int main(){

    int a, b, c, menor; 

    printf("Primeiro valor: ");
    scanf("%i", &a);

    printf("Segundo valor: ");
    limpar_entrada();
    scanf("%i", &b);

    printf("Terceiro valor: ");
    limpar_entrada();
    scanf("%i", &c); 

    if (a < b && a < c) {
        menor = a;

    }
    else if (b < c){
        menor = b;
    }
    else {
        menor = c;
    }
    

    printf("MENOR = %i", menor); 
    

    return 0;
}

File: logica-de-programacao/secao-13/exercicios/menor_de_tres.py
n1 = int(input("Primeiro valor: "))
n2 = int(input("Segundo valor: "))
n3 = int(input("Terceiro valor: "))

if n1 < n2 and n1 < n3:
    menor = n1
elif n2 < n1 and n2 < n3:
    menor = n2
else:
    menor = n3

print(f"Menor = {menor}")
